packages/docs/pages/index.vue

```diff
@@ -5,7 +5,7 @@
       class="base-layout__header"
     />
     <div class="base-layout__main">
-      <Sidebar v-if="isSidebarVisible" :minimized="!isSidebarVisible" />
+      <Sidebar v-if="isSidebarVisible" :minimized="!isSidebarVisible" :items="navigationItems" />
```

Comment:
What's the reason to keep v-if here?
I see the comment below, but it doesn't tell much.

Reply:
We should rework it when we will implement iconset handling and will show icons when sidebar is hidden.